pkg/process/config/config_nix_test.go

```diff
@@ -56,20 +56,13 @@ func TestAgentConfigYamlEnc(t *testing.T) {
 	config.Datadog = config.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
 	defer restoreGlobalConfig()
 	// Secrets settings are initialized only once by initConfig in the agent package so we have to setup them
+	config.InitConfig(config.Datadog)
 	config.Datadog.Set("secret_backend_timeout", 15)
 	config.Datadog.Set("secret_backend_output_max_size", 1024)
 
-	assert := assert.New(t)
-
-	agentConfig, err := NewAgentConfig(
-		"test",
-		"./testdata/TestDDAgentConfigYamlEnc.yaml",
-		"",
-	)
-	assert.NoError(err)
-
+	agentConfig := loadAgentConfigForTest(t, "./testdata/TestDDAgentConfigYamlEnc.yaml", "")
```

Comment:
Looks like you are dropping the `assert := assert.New(t)` line here, so that's why line 65 gets changed.